lang/test.lang

```diff
@@ -0,0 +1,74 @@
+mario[] -> static
```

Comment:
`test.asm` doesn't have a `main` label in it and the first line is `GOTO main`

Reply:
ye my bad there was no main() in test.lang